core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -114,6 +114,15 @@ object NextTurnAutomation {
 
     /** allow AI to spend money to purchase city-state friendship, buildings & unit */
     private fun useGold(civInfo: CivilizationInfo) {
+        if (civInfo.getHappiness() > 0) {
```

Comment:
Shouldn't a check per CS be better? AI knowns 12 CS and has tons of gold -> marries all 12 in 1 turn -> -36 Happy or whatever... Or limit to 1 per turn for the AI

Reply:
1 per turn max sounds good.